bench/wtperf/runners/modify-read-btree.wtperf

```diff
@@ -0,0 +1,12 @@
+conn_config="cache_size=20G"
+table_config="type=file"
+key_sz=100
+value_sz=10240
+value_sz_max=1048576
```

Comment:
is there a reason we need a range of sizes for keys and values?

Reply:
We want to append to the values to change the value size.